app/uk/gov/nationalarchives/omega/editorial/forms/CredentialsFormProvider.scala

```diff
@@ -0,0 +1,21 @@
+package uk.gov.nationalarchives.omega.editorial.forms
+
+import play.api.data.Form
+import play.api.data.Forms.{ mapping, text }
+import play.api.i18n.{ Lang, MessagesApi }
+import uk.gov.nationalarchives.omega.editorial.models.Credentials
+
+object CredentialsFormProvider {
+
+  val editorialUsername = scala.util.Properties.envOrElse("CTD_EDITORIAL_USERNAME", "1234")
+  val editorialPassword = scala.util.Properties.envOrElse("CTD_EDITORIAL_PASSWORD", "1234")
```

Comment:
These should be marked `private` so that are not accessible outside of the object.